snippets/views-haml.cson

```diff
@@ -0,0 +1,184 @@
+'.text.haml':
```

Comment:
this new file is your views snippets (erb) but in haml sintax...

Note: snippets don't have `<%  %>` tags